themes/default/content/blog/pulumi-release-notes-69/index.md

```diff
@@ -0,0 +1,86 @@
+meta_image: meta.png
+
+# At least one author is required. The values in this list correspond with the `id`
+# properties of the team member files at /data/team/team. Create a file for yourself
+# if you don't already have one.
+authors:
+    - meagan-cojocar
+
+# At least one tag is required. Lowercase, hyphen-delimited is recommended.
+tags:
+    - features
+
+---
+
+The team has been busy releasing new features and improvements in the last 3 weeks. Read on to learn about what's new in this release!
+
+- Pulumi CLI and core technologies
+  - [`pulumi import` Improvements](#pulumi-import-improvements)
+  - [Add `RetainOnDelete` as a resource option](#add-retainondelete-as-a-resource-option)
+  - [New `pulumi state rename` command](#new-pulumi-state-rename-command)
+  - [Default `pulumi plugin install` to the latest version](#default-pulumi-plugin-install-to-the-latest-version)
+  - [Add console output in non-interactive mode](#add-console-output-in-non-interactive-mode)
+  - [Support `pulumi cancel` for filestate backends](#support-pulumi-cancel-for-filestate-backends)
+
+<!--more-->
+
+
+## Pulumi CLI and core technologies
+
+### `pulumi import` Improvements
+
+Last year, we introduced a new Pulumi import feature that allows you to import existing infrastructure into your Pulumi program. We’ve listened to feedback and delivered a plethora of updates and fixes to streamline the import experience; to make it more useful, more convenient, and more powerful. Not only can this feature bring a resource into your Pulumi state file, but it can also generate the source code for your Pulumi program too. 
+
+You can now use the `pulumi import` command to import using all input fields instead of only the ones flagged as required, which will result in less check failures. If there are check failures, they are now treated as a warning not an error. In addition, we added an optional flag to `pulumi import` that allows users to skip the code generation step of import.
+
+Learn more in the [Improved Import blog post]({{< relref "/blog/changes-to-import" >}}), [import improvements GitHub issue](https://github.com/pulumi/pulumi/issues/9134) and the [disable codegen GitHub issue](https://github.com/pulumi/pulumi/issues/9134). 
+
+
+### Add `RetainOnDelete` as a resource option
+
+Pulumi is frequently used to manage the entire lifecycle of a resource, from creation, to updates, to replacement, to deletion. However, there are some cases where it is important to ensure that a resource’s life can extend beyond the lifetime of the Pulumi program that created it. To support these use cases, Pulumi now supports a new resource option `RetainOnDelete` which allows a resource to be retained in a cloud provider even after it is deleted from the Pulumi stack it is part of.
+
+Learn more in the [RetainOnDelete blog post]({{< relref "/blog/retainondelete" >}}) and in the [RetainOnDelete GitHub issue](https://github.com/pulumi/pulumi/issues/7747).
```

Comment:
What do you think about linking to [the RetainOnDelete docs](https://www.pulumi.com/docs/intro/concepts/resources/options/retainondelete/) as well here?